worlds/sc2/__init__.py

```diff
@@ -335,3 +435,35 @@ def get_plando_locations(multiworld: MultiWorld, player) -> List[str]:
             plando_locations.append(plando_setting_location)
 
     return plando_locations
+
+def fill_pool_with_kerrigan_levels(multiworld: MultiWorld, player: int, item_pool: List[Item]):
+    total_levels = get_option_value(multiworld, player, "kerrigan_level_item_sum")
+    if get_option_value(multiworld, player, "kerriganless") > 0 \
+        or total_levels == 0:
+        return
+    
+    def add_kerrigan_level_items(level_amount: int, item_amount: int):
+        name = f"{level_amount} Kerrigan Level"
+        if level_amount > 1:
+            name += "s"
+        for _ in range(item_amount):
+            item_pool.append(create_item_with_correct_settings(player, name))
+
+    sizes = [70, 35, 14, 10, 7, 5, 2, 1]
+    option = get_option_value(multiworld, player, "kerrigan_level_item_distribution")
+    if option < 2:
+        distribution = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+        if option == 0: # vanilla
+            distribution = [32, 0, 0, 1, 3, 0, 0, 0, 1, 1]
+        elif option == 1: # smooth
+            distribution = [0, 0, 0, 1, 1, 2, 2, 2, 1, 1]
+        for tier in range(len(distribution)):
+            add_kerrigan_level_items(tier + 1, distribution[tier])
```

Comment:
Use the option symbolic names. There are also other spots of that:
```python
    if mission_order_type == MissionOrder.option_vanilla:
```

Reply:
Done